.circleci/config.yml

```diff
@@ -223,148 +223,157 @@ jobs:
           root: packages
           paths:
             - docs-site/coverage
+  workflow_disabled:
+    docker: *docker
+    steps:
+      - run: echo "this workflow is disabled"
 
 workflows:
```

Comment:
Can we just delete this?

Reply:
We could but it will report errors unless we remove the whole project from Circle which means losing all history etc.  I didnt want to do that yet until we are completely happy with the new solution so have left the circle configured to  run a minimal workflow task (it should not trigger anyway as we will no longer be committing to the latest branch).

Reply:
The main point here was favouring deleting over commenting out as it's available in version control anyway.